modules/end-user-guide/pages/git-credential-store.adoc

```diff
@@ -1,25 +1,48 @@
-:navtitle: Git credential store
+:navtitle: Creating a {orch-name} Secret for a Git credentials store
 :keywords: user-guide, configuring, user, secrets
 :page-aliases: 
 
-[id="mounting-a-git-credential-store-into-a-workspace-container_{context}"]
-= Mounting a Git credentials store into a workspace container
+[id="mounting-a-git-credential-store-into-workspace-containers_{context}"]
+= Creating a {orch-name} Secret for a Git credentials store
 
-This section describes how to mount Git credentials store as secret from the user's {orch-namespace} into the file in single-workspace or multiple-workspace containers of {prod-short}.
+This section describes how to create a Secret containing a Git credentials store in the user's {orch-namespace}. The Git credentials store specified in the Secret is mounted as a file in the {devworkspace}'s containers.
 
 .Prerequisites
 
 * A running instance of {prod-short}.
+* You have CLI or GUI access to the {platforms-name} cluster of your organization's {prod-short} instance.
 
 .Procedure
-. Prepare Git credential file in the link:https://git-scm.com/docs/git-credential-store#_storage_format[Storage format].
-. Encode content of the file to the base64 format.
-. Create a new {platforms-name} secret in the {platforms-name} {orch-namespace} where a {prod-short} workspace will be created.
+. Prepare a Git credential file in the link:https://git-scm.com/docs/git-credential-store#_storage_format[Storage format].
 
-* The labels of the secret that is about to be created must match the set of labels configured in `che.workspace.provision.secret.labels` property of {prod-short}. The default labels are:
+. Encode content of the file to the base64 format.
 
-* `app.kubernetes.io/part-of: che.eclipse.org` 
-* `app.kubernetes.io/component: workspace-secret`:
+. Create a new {platforms-name} secret in the user's {orch-namespace}.
 +
-include::example$snip_{project-context}-git-credential-file-as-kubernetes-secret-into-the-file.adoc[leveloffset=+1]
+[source,yaml,subs="+quotes,+attributes,+macros"]
+----
+apiVersion: v1
+kind: Secret
+metadata:
+  name: git-credentials-secret
+  labels:
+    controller.devfile.io/git-credential: 'true'                 <1>
+    controller.devfile.io/watch-secret: 'true'
+  annotations:
+    controller.devfile.io/mount-path: __<path_in_container>__        <2>
+data:
+  credentials: __<base64_content_of_the_git_credentials_store>__     <3>
+----
++
+<1> The `controller.devfile.io/git-credential` label marks the secret as containing Git credentials. All Git credential secrets in the same {orch-namespace} will be merged into a single secret that would be mounted to the {devworkspace} containers. All the original secrets are kept in-tact.
```

Comment:
```suggestion
<1> The `controller.devfile.io/git-credential` label marks the secret as containing Git credentials. All Git credential secrets in the same {orch-namespace} will be merged into a single secret that will be mounted to the {devworkspace} containers. All the original secrets are kept intact.
```

Reply:
`All Git credential secrets in the same {orch-namespace} will be merged into a single secret that will be mounted to the {devworkspace} containers.`
-- `All Git credential secrets` of all Che users or just of this particular Che user?

Reply:
In https://www.eclipse.org/che/docs/che-7/end-user-guide/mounting-a-secret-as-a-file-or-an-environment-variable-into-a-workspace-container/#mounting-a-git-credential-store-into-a-workspace-container_che, it looks like we mounted as a file previously.

Reply:
> All Git credential secrets in the same {orch-namespace} 

It means Git credential secrets of this particular Che user, since a user's namespace would only contain that user's secrets.

> Question: If this secret is to be mounted as a file, then ...

Yes, the secret is mounted as a file, but the file name itself (`credentials`) cannot change. Right @amisevsk ?

Reply:
> All Git credential secrets in the same {orch-namespace} will be merged into a single secret that will be mounted to the {devworkspace} containers.

Means exactly what it says -- any secrets with the git-credential label will be merged and mounted to all workspaces, regardless of who created them. If I access David's namespace and add my git credentials secret, it will be mounted to any workspaces started in that namespace.

The name of the file (credentials) is hardcoded in DWO for convenience. The user cannot choose the name of file, as it is named after the secret data key that stores the value. Using e.g. `che.eclipse.org/mount-path: /home/theia/.git-credentials` results in a file
```
/home/theia/.git-credentials/credentials
```
being mounted to workspace containers.

Reply:
The `credentials` filename comes from the secret definition above:
```
credentials: __<base64_content_of_the_git_credentials_store>__
```